docs/K4-v2.md

```
@@ -0,0 +1,150 @@
+2. **K4 Ciphertext Matrix Monitor (`#cipher-matrix-crt`)**
+* **Visual:** Amber CRT Nixie-style segment modules displaying the 97-character K4 Ciphertext string.
+* **Functionality:** Highlights anchored plaintext segments dynamically in green/red:
+* `EAST` (pos 22–25)
+* `NORTHEAST` (pos 26–34)
+* `BERLIN` (pos 63–68)
+* `CLOCK` (pos 69–73)
+
+
+
+
+3. **System Status & Vault Access (`#sys-status-panel`)**
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)